src/components/UserCatigories.tsx

```diff
@@ -0,0 +1,73 @@
+import "./UserCategories.scss";
+import React from "react";
+import classNames from "classnames";
+
+type CategoryImageProps = {
+    imagePreview: string;
+    opacity: boolean;
+};
+const CategoryImage: React.FC<CategoryImageProps> = (props) => {
+  return (
+    <div className={props.opacity ? "category__image-transparent" : ""}>
+      <img src={props.imagePreview} alt='category'/>
+    </div>
+  );
+};
+
+type CategoryNameProps = {
+  name: string;
+  className: string;
+};
+
+const CategoryName: React.FC<CategoryNameProps> = (props) => {
+  return (
+    <div className={props.className}>{props.name}</div>
+  );
+};
+
+type CategoryInfoProps = {
+  text: string;
+};
+
+const CategoryInfo: React.FC<CategoryInfoProps> = (props) => {
+  return (
+    <div className="category__info">{props.text}</div>
+  );
+};
+
+type UserCategoriesProps = {
+  // id: number; use only for admin
+  imagePreview: string;
+  name: string;
+  text: string;
+  bgColor: "blue" | "pink" |"grey"| "orange";
+};
+
+const UserCategories: React.FC<UserCategoriesProps> = (props) => {
```

Comment:
Please, rename to singular form: UserCategory, or CategoryCard

Reply:
renamed